packages/near-api-js/test/account.test.js

```diff
@@ -158,6 +158,27 @@ describe('with deploy contract', () => {
         expect(await workingAccount.viewFunction(contractId, 'getValue', {})).toEqual(setCallValue);
     });
 
+    test('make function calls via txn', async () => {
+        const result = await workingAccount.viewFunction(
+            contractId,
+            'hello', // this is the function defined in hello.wasm file that we are calling
+            { name: 'trex' }
+        );
+        expect(result).toEqual('hello trex');
+
+        const setCallValue = testUtils.generateUniqueString('setCallPrefix');
+        const result2 = await workingAccount
+            .createTransaction(contractId)
+            .functionCall('setValue', { value: setCallValue })
```

Comment:
Should we add more tests here? We're only really testing `functionCall` and `signAndSend` but I feel there is much more logic to cover?

Reply:
Added coverage for the same cases as `Account` class 👍